packages/cli/src/util/metrics.ts

```diff
@@ -16,10 +15,15 @@ export const shouldCollectMetrics =
 
 export const metrics = (): ua.Visitor => {
   const token =
-    typeof config.token === 'string' ? config.token : platform() + release();
-  const salt = userInfo().username;
+    typeof config.token === 'string'
+      ? config.token
+      : process.platform + process.arch;
+  const salt =
+    (process.env.USER || '') +
+    (process.env.LANG || '') +
+    (process.env.SHELL || '');
   const hash = crypto
-    .pbkdf2Sync(token, salt, 1000, 64, 'sha512')
+    .pbkdf2Sync(token, salt, 100, 64, 'sha512')
```

Comment:
I reduced the iterations here for speed because this doesn't need to be cryptographically secure (its being truncated to 24 chars anyway)